cmd/trade.go

```diff
@@ -93,20 +97,20 @@ func init() {
 		utils.CheckConfigError(botConfig, e, *botConfigPath)
 		e = botConfig.Init()
 		if e != nil {
-			log.Println()
-			log.Fatal(e)
+			l.Info("")
```

Comment:
the first line of the `tradeCmd.Run` function should be `l := logger.MakeBasicLogger()`

this will eliminate the global variable issue

Reply:
I had to put it at the beginning `func init()` because it needs to be before `validateCliParams`